gobblin-yarn/src/test/java/org/apache/gobblin/yarn/YarnServiceTest.java

```diff
@@ -316,6 +316,7 @@ private static HelixManager getMockHelixManager(Config config) {
       Mockito.when(helixManager.getClusterName()).thenReturn(config.getString(GobblinClusterConfigurationKeys.HELIX_CLUSTER_NAME_KEY));
 
       Mockito.when(helixManager.getHelixDataAccessor()).thenReturn(helixDataAccessor);
+      Mockito.when(helixManager.getMetadataStoreConnectionString()).thenReturn("stub");
```

Comment:
YarnService test doesn't run without this mock. I accidentally broke this test in https://github.com/apache/gobblin/pull/3561 but this wasn't caught because this test doesn't run in CI due to flakiness and slowness